docs/example-scenario/data/data-warehouse.md

```diff
@@ -0,0 +1,122 @@
+* Consolidating data from numerous heterogenous data sources to a cloud-scale data platform
+* Transforming source data into a common taxonomy and structure for consistency and comparability.   
+* Leveraging highly parallelized data loading to support thousands of incentive programs without the expense of on-premises infrastructure deployment and maintenance
+* Significantly improving the ratio of time spent gathering and transforming data vs. time spent analyzing data
+
+This example scenario is relevant to organizations that want to an integrated data pipeline from multiple data sources into a cloud-scale platform for advanced analysis and business intelligence capabilities. Potential applications include sales and marketing, e-commerce, healthcare, government, or other industry solutions involving large volumes of data that can deliver high business value through analysis. 
+
+Using managed Azure services such as Data Factory and SQL Data Warehousecan significantly reduce costs by leveraging Microsoft's expertise in globally distributed cloud-scale data storage and analysis. If you have additional data service needs, you should review the list of available [fully managed intelligent database services in Azure][product-category].
+
+## Potential use cases
+
+Consider this solution for the following use cases:
+
+* Creation and incremental revision of a data warehouse that serves as a single source of truth for your organization's data.
+* Integration of relational data with binary data or other non-relational datasets.
+* Analysis of large datasets by business data analysts through a well-defined semantic model and robust visualization tools.
+
+## Architecture
+
+![Architecture for a data warehouse scenario in Azure][architecture]
+
+This example scenario is based on the [Azure automated enterprise BI reference architecture](/azure/architecture/reference-architectures/data/enterprise-bi-adf). The data flows through the solution as follows:
+
+1. Data from each data source is periodically exported into Azure Blob storage, which serves as a staging area for the source data. Each export contains only the data that has changed since the previous export. 
+2. Data Factory incrementally loads the staged data from Blob storage into staging tables in SQL Data Warehouse. Data cleansing and transformation happens at this stage, and Polybase can be used to parallelized the process for large volumes of data.
+3. After a new batch of data is loaded into the warehouse, a previously created Analysis Services tabular model is refreshed. This provides a semantic model that simplifies the analysis of business data and relationships by business analysts.
+4. Business analysts use Power BI to analyze data in the warehouse via the Analysis Services semantic model.
+
+### Components
```

Comment:
I'll think about this one. For the data sources in this case study, we don't really have a defined rationale for its use beyond it being a repository of data that we want to include in the data warehouse.

Reply:
I wonder if we shouldn't even include line items for the data source components, because it's really just illustrating several options out of many data sources that a customer might have.